spec/observables/defer-spec.ts

```diff
@@ -72,4 +72,16 @@ describe('Observable.defer', () => {
     expectObservable(e1, unsub).toBe(expected);
     expectSubscriptions(source.subscriptions).toBe(sourceSubs);
   });
+
+  type(() => {
```

Comment:
it seems like it would be better just to work that into the TypeScript of our actual tests. Since those will cause the compilation of the spec file themselves to fail and thus cause the tests to fail. Do we really need an additional test that doesn't seem to do anything once it's transpiled?